thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js

```diff
@@ -346,7 +346,9 @@ export default Ember.Controller.extend({
   callReplayStart(functionId, startTime, endTime) {
     const granularity = this.get('graphConfig.granularity').toLowerCase();
     const speedUp = granularity.includes('hour') || granularity.includes('day');
```

Comment:
speedUp is "true" for minutely metrics, speedUp is "false" for hourly/daily metrics, is the logic correct here?